common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java

```diff
@@ -1547,6 +1547,36 @@ public static String computeMaxHostBrokerProtocol() {
 
     }
 
+    public static final class IntuneContentProviderCall {
```

Comment:
Is this meant to be in this PR?

Reply:
Yes, these are @tanmaymanolkar1's changes from #1513